cmd/template/cluster/flag.go

```diff
@@ -56,6 +59,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.Owner, flagOwner, "", "Tenant cluster owner organization.")
 	cmd.Flags().StringVar(&f.Region, flagRegion, "", "Installation region (e.g. eu-central-1).")
 	cmd.Flags().StringVar(&f.Release, flagRelease, "", "Tenant cluster release.")
+	cmd.Flags().StringSliceVar(&f.Label, flagLabel, []string{}, "Tenant cluster label.")
```

Comment:
```suggestion
	cmd.Flags().StringSliceVar(&f.Label, flagLabel, nil, "Tenant cluster label.")
```

Reply:
`nil` is actually a slice's zero value (the value it has by default). A `nil` slice has a `nil` backing array. This means less computation. Also, the Go compiler applies several optimizations for `nil` slices, whereas empty slices can only be optimized at run time.

A more visual example (how a slice data structure looks in go)

* **nil slice**
```go
{
	capacity: 0
	length: 0
	backingArray: nil
}
```

* **empty slice**
```go
{
	capacity: 0
	length: 0
	backingArray: []
}
```

The actual performance difference won't be noticeable in this case, but it's good practice to use `nil` slices in general.

All built-in slice functions accept `nil` slices. That's actually the reason why we get a slice's length by using `len(slice)`, instead of `slice.length`. The only issue is when you marshal to JSON (a `nil` slice is serialized as `null`, as opposed to an empty array)

Reply:
:pray: many thanks thats a good explanation!